Sources/MessagingInApp/Gist/Gist.swift

```diff
@@ -146,4 +155,21 @@ public class Gist: GistDelegate {
     func removeMessageManager(instanceId: String) {
         messageManagers.removeAll(where: { $0.currentMessage.instanceId == instanceId })
     }
+
+    // Inline messages
+
+    func getInlineMessages(forElementId elementId: String) -> [Message] {
+        messageQueueManager.getInlineMessages(forElementId: elementId)
+    }
```

Comment:
Only logic added for inline. Rest of code modified is to make the code testable.